Box.V2/Managers/BoxMetadataManager.cs

```diff
@@ -66,7 +66,7 @@ public BoxMetadataManager(IBoxConfig config, IBoxService service, IBoxConverter
         /// <param name="metadata">Metadata to create</param>
         /// <param name="scope">Scope name. Currently, only the enterprise scope is supported</param>
         /// <param name="template">Metadata template name</param>
-        /// <returns>A Dictionary of key:value pairs representing the metadata.</returns>
+        /// <returns>An instance of the template that includes key:value pairs defined by a user or application.</returns>
         public async Task<Dictionary<string, object>> CreateFolderMetadataAsync(string folderId, Dictionary<string, object> metadata, string scope, string template)
         {
             return await CreateMetadata(_config.FoldersEndpointUri, folderId, metadata, scope, template);
```

Comment:
why there is no validation like below?
            folderId.ThrowIfNullOrWhiteSpace("folderId");
            scope.ThrowIfNullOrWhiteSpace("scope");
            template.ThrowIfNullOrWhiteSpace("template");